adlfs/spec.py

```diff
@@ -73,6 +73,12 @@
 _SOCKET_TIMEOUT_DEFAULT = object()
```

Comment:
We should also make sure to add a changelog entry for this update here: https://github.com/fsspec/adlfs/blob/main/CHANGELOG.md